src/integrationTest/groovy/wooga/gradle/build/unity/ios/tasks/PodInstallTaskSpec.groovy

```diff
@@ -0,0 +1,95 @@
+package wooga.gradle.build.unity.ios.tasks
+
+import spock.lang.Requires
+
+/**
+ * The test examples in this class are not 100% integration/functional tests.
+ *
+ * We can't run the real pod install as this would bring to much overhead at the moment.
+ * We only test the invocation of podß and its parameters.
```

Comment:
Typo?

Reply:
narf. I have a new keyboard and constantly hit the ALT key.

Reply:
ALT-S instead of CMD-S. It's a windows keyboard and I did not remap the keys yet.